solarwinds_apm/distro.py

```diff
@@ -99,112 +86,69 @@ def _get_token_from_service_key(self):
             return None
         return key_parts[0]
 
-    def _configure_logs_export_env_defaults(
+    def _configure_otlp_env_defaults(
         self,
+        signal_type: str,
         header_token: str,
         otlp_protocol: str,
     ) -> None:
-        """Configure env defaults for OTLP logs signal export by HTTP or gRPC to SWO"""
-        if otlp_protocol in _EXPORTER_BY_OTLP_PROTOCOL:
-            environ.setdefault(OTEL_EXPORTER_OTLP_LOGS_PROTOCOL, otlp_protocol)
-            environ.setdefault(
-                OTEL_LOGS_EXPORTER, _EXPORTER_BY_OTLP_PROTOCOL[otlp_protocol]
-            )
+        """Configure env defaults for OTLP signal export by HTTP or gRPC to SWO"""
+        protocol_env_var = f"OTEL_EXPORTER_OTLP_{signal_type.upper()}_PROTOCOL"
+        exporter_env_var = f"OTEL_{signal_type.upper()}_EXPORTER"
+        endpoint_env_var = f"OTEL_EXPORTER_OTLP_{signal_type.upper()}_ENDPOINT"
+        headers_env_var = f"OTEL_EXPORTER_OTLP_{signal_type.upper()}_HEADERS"
+
+        environ.setdefault(protocol_env_var, otlp_protocol)
+        environ.setdefault(
+            exporter_env_var, _EXPORTER_BY_OTLP_PROTOCOL[otlp_protocol]
+        )
+        environ.setdefault(
+            endpoint_env_var,
+            f"{INTL_SWO_DEFAULT_OTLP_COLLECTOR}/v1/{signal_type}",
+        )
+        if header_token:
```

Comment:
A `debug` log is emitted by the caller if no header_token, see [this comment](https://github.com/solarwinds/apm-python/pull/499#discussion_r1937783816)